.luacheckrc

```diff
@@ -0,0 +1,83 @@
+codes = true
+std = "min"
+self = false
+
+globals = {
+	"getfenv",
+	"setfenv",
+	"unpack",
+}
+
+include_files = {
+	"**/*.lua",
+	"package/**/files/lib/gluon/ebtables/*",
+	"package/**/luasrc/**/*",
+}
+
+files["**/check_site.lua"] = {
+	globals = {
+		"alternatives",
+		"extend",
+		"in_domain",
+		"in_site",
+		"need",
+		"need_alphanumeric_key",
+		"need_array",
+		"need_array_of",
+		"need_boolean",
+		"need_chanlist",
+		"need_domain_name",
+		"need_number",
+		"need_one_of",
+		"need_string",
+		"need_string_array",
+		"need_string_array_match",
+		"need_string_match",
+		"need_table",
+		"need_value",
+		"obsolete",
+		"table_keys",
+		"this_domain",
+	},
+}
+
+files["package/**/files/lib/gluon/ebtables/*"] = {
+	globals = {
+		"site",
```

Comment:
This should not be global, it's just another missing local.

Reply:
You fixed it in the ebtables file, i removed the global.